client/components/support-info/index.jsx

```diff
@@ -1,52 +1,59 @@
 import { localize } from 'i18n-calypso';
 import PropTypes from 'prop-types';
-import { Component } from 'react';
 import ExternalLink from 'calypso/components/external-link';
 import InfoPopover from 'calypso/components/info-popover';
 
 import './style.scss';
 
-class SupportInfo extends Component {
-	static propTypes = {
-		text: PropTypes.string,
-		link: PropTypes.string,
-		position: PropTypes.string,
-		privacyLink: PropTypes.oneOfType( [ PropTypes.string, PropTypes.bool ] ),
-	};
-
-	static defaultProps = {
-		text: '',
-		link: '',
-		privacyLink: '',
-	};
-
-	render() {
-		const { text, link, position, privacyLink, translate } = this.props;
-		const actualPrivacyLink =
-			! privacyLink && privacyLink !== false && link ? link + '#privacy' : privacyLink;
-
-		return (
-			<div className="support-info">
-				<InfoPopover position={ position || 'left' } screenReaderText={ translate( 'Learn more' ) }>
-					{ text + ' ' }
-					{ link && (
-						<span className="support-info__learn-more">
-							<ExternalLink href={ link } target="_blank" rel="noopener noreferrer">
-								{ translate( 'Learn more' ) }
-							</ExternalLink>
-						</span>
-					) }
-					{ actualPrivacyLink && (
-						<span className="support-info__privacy">
-							<ExternalLink href={ actualPrivacyLink } target="_blank" rel="noopener noreferrer">
-								{ translate( 'Privacy Information' ) }
-							</ExternalLink>
-						</span>
-					) }
-				</InfoPopover>
-			</div>
-		);
+function SupportInfo( { text, link, position, translate, privacyLink } ) {
+	function makePrivacyLink() {
```

Comment:
Usually helper functions are put outside the component, so they're not recreated at every render.
```suggestion
	function makePrivacyLink(link, previewLink) {
```